pkg/repository/v1_repository.go

```diff
@@ -94,12 +95,14 @@ func (r *V1Repository) UpdateComponents(specs []ComponentSpec) error {
 		}
 
 		if spec.Nightly {
-			spec.Version = manifest.Nightly
+			spec.Version = version.NightlyVersion
```

Comment:
We already have nightly builds in new filename format, maybe this is no longer needed? (i'm not sure)